README.md

```diff
@@ -175,15 +175,6 @@ It can also be configured to custom get_input field name, but in this case, user
 
     curl -X "POST" -F "inputs_file=@path/to/file.yaml" localhost:5000/deploy/567858fc-a1e8-43b4-91f5-cb04ec8be90b
 
-### Using non default user
```

Comment:
I would leave and extend this example but change the text so that is clearly stated that the user needs to supply OS_* values in the inputs file in order to access OpenStack environments..